module-common/src/main/resources/application-common-local.yml

```
@@ -0,0 +1,3 @@
+jwt:
+  secret: 1edUaaGpfy135hntJVinhw7gM0nNfM2SPfbDiWEsQtbBYtR8VQxyRP2ooyOm/3tolC/Ok3gW3hr8Haztvs7XLQ==
+  expired: 3600000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)